### PR TITLE
Added autoloading of .env files

### PR DESCRIPTION
**Pull request by ADRFranklin**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/Southclaws/sampctl/types"
 )
 
+import _ "github.com/joho/godotenv/autoload"
```
Comment (Southclaws) [inline main.go]: why is this a separate import statement?

Comment (ADRFranklin) [reply main.go]: Sorry I was trying to work out why it wasn't working for me, and it worked like that so didn't think too much into it. I'll move it back into the import list

File: main.go [hunk 2]
```diff
@@ -8,10 +8,11 @@ import (
 	"time"
 
 	"github.com/Masterminds/semver"
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/github"
```
Comment (Southclaws) [inline main.go]: nice tab ;)

Comment (ADRFranklin) [reply main.go]: weird that doesn't show up in my editor :O

Comment (ADRFranklin) [reply main.go]: oh never mind, restarted vscode and it was there

**Pull request merged**